File: app/src/main/java/foocafe/org/foocafe/entities/Event.java
package foocafe.org.foocafe.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Event implements Serializable {
    @SerializedName("title")
    public String title;

    @SerializedName("subtitle")
    public String subtitle;

    @SerializedName("image")
    public String image;

    @SerializedName("date")
    public String date;

    @SerializedName("start")
    public String time;

    @SerializedName("description")
    public String description;

    @SerializedName("checkmark")
    public Boolean checkmark = false;

    @SerializedName("id")
    public String event;

    @SerializedName("url")
    public String url;
}

File: app/src/main/java/foocafe/org/foocafe/entities/Event.kt
package foocafe.org.foocafe.entities

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Event : Serializable {
    @SerializedName("title")
    var title: String? = null

    @SerializedName("subtitle")
    var subtitle: String? = null

    @SerializedName("image")
    var image: String? = null

    @SerializedName("date")
    var date: String? = null

    @SerializedName("start")
    var time: String? = null

    @SerializedName("description")
    var description: String? = null

    @SerializedName("checkmark")
    var checkmark: Boolean? = false

    @SerializedName("id")
    var event: String? = null

    @SerializedName("url")
    var url: String? = null
}